content/about/funding.md

```diff
@@ -0,0 +1,94 @@
+---
+$title@: Funding
+$order: 7
+---
+
+This page lists currently active funding for our projects
+
+#### Project Title: Synaptomes of Mouse and Man
+**R01NS092474 (Smith)**: 9/30/2014 – 6/30/2019  
```

Comment:
we should probably add a link to each of the funding pages for each of these grants.

for instance, for synaptomes of mouse and man, http://grantome.com/grant/NIH/R01-NS092474-01

note, the synaptomes grant is about to end.

Reply:
Cool, will do. I can add a hyperlink into either the grant title or the project title.

Reply:
not urgent

Reply:
ok, to do later then. Will add to the issue.